src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,39 @@
+import { Footer as HDSFooter } from 'hds-react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+const T_PATH = 'components.footer';
+
+const Footer = (): React.ReactElement => {
+  const { t } = useTranslation();
+  const FOOTER_COLOR = 'var(--color-white)';
+  return (
+    <HDSFooter
+      theme={{
+        '--footer-background': 'var(--color-black)',
+        '--footer-color': FOOTER_COLOR,
+        '--footer-divider-color': FOOTER_COLOR,
+        '--footer-focus-outline-color': FOOTER_COLOR,
+      }}>
```

Comment:
Could use [dark theme](https://hds.hel.fi/components/footer/code#color-variations) here instead of this.
```suggestion
    <HDSFooter theme="dark">
```